codecparsers/bitReader_unittest.cpp

```diff
@@ -48,12 +48,12 @@ BITREADER_TEST(NoOverflow)
     std::vector<uint8_t>& data = largeData();
     const uint32_t nBytes = data.size();
 
-    ASSERT_EQ(0, nBytes % (1024 * 1024 * 512));
+    ASSERT_EQ(0u, nBytes % (1024 * 1024 * 512));
```

Comment:
kind of wired, if i apply this patch
```
diff --git a/decoder/vaapidecoder_h264.cpp b/decoder/vaapidecoder_h264.cpp
index 646f500..66c8b70 100644
--- a/decoder/vaapidecoder_h264.cpp
+++ b/decoder/vaapidecoder_h264.cpp
@@ -1501,7 +1501,14 @@ YamiStatus VaapiDecoderH264::decodeCurrent()
     YamiStatus status = YAMI_SUCCESS;
     if (!m_currPic)
         return status;
-
+    int t = -1;
+    unsigned int s = 0;
+    if (t == 0u) {
+        printf("aaaaaa");
+    }
+    if (t == s) {
+        printf("failed");
+    }
     if (!m_currPic->decode()) {
         ERROR("decode %d failed", m_currPic->m_poc);
         // ignore it to let application continue to decode the next frame

```
gcc only report error for 
`if (t == s) {`
not for
` if (t == 0u) {`

```
make[2]: Entering directory `/home/thomas/projects/yami/libyami/decoder'
  CXX      libyami_decoder_la-vaapidecoder_h264.lo
vaapidecoder_h264.cpp: In member function 'YamiStatus YamiMediaCodec::VaapiDecoderH264::decodeCurrent()':
vaapidecoder_h264.cpp:1509:14: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
     if (t == s) {
              ^
cc1plus: all warnings being treated as errors
make[2]: *** [libyami_decoder_la-vaapidecoder_h264.lo] Error 1
```